src/strategies/spidStrategy.ts

```diff
@@ -129,4 +134,27 @@ const spidStrategy = async (
   );
 };
 
+/**
+ * Reads dates information in x509 certificate and logs remaining time to its expiration date.
+ * @param samlCert x509 certificate as string
+ */
+function logSamlCertExpiration(samlCert: string): void {
+  try {
+    const tempDir = fs.mkdtempSync("saml-cert");
+    const certPath = `${tempDir}/samlCert.pem`;
+    fs.writeFileSync(certPath, samlCert);
```

Comment:
you [can read the cert from a string](https://www.npmjs.com/package/x509#usage) there's no need to write to a temp file

Reply:
fixed in 42c0a16